### PR TITLE
Feature 18: Add page views for language page

### PR DESCRIPTION
**Pull request by PhotoNomad0**
Part of: https://github.com/unfoldingWord-dev/door43.org/issues/18
And also addresses: https://github.com/unfoldingWord-dev/door43.org/issues/635

on language page load, updates page view count.

Live on http://test-door43.org.s3-website-us-west-2.amazonaws.com/en/

### CODE REVIEW

**Comment by PhotoNomad0** [comment]
WIP

**Comment by PhotoNomad0** [comment]
Ready for review.

File: js/application.js [hunk 1]
```diff
@@ -95,3 +97,105 @@ function scrollToPosition(hash) {
...
+function buildAnchor(href, text) {
+    return "<a href=\"" + href + "\">" + text + "</a>";
+}
+
+/**
+ * for missing language page update options on page
+ * @param lang_code
+ * @param subPath
+ */
+function changeMissingtextForLanguageCode(lang_code, subPath) {
```
Comment (richmahn) [inline js/application.js]: Uppercase T in 'text'

File: js/application.js [hunk 2]
```diff
@@ -95,3 +97,105 @@ function scrollToPosition(hash) {
...
+/**
+ * if href if 404 page and a language page, update displayed text with more options
+ * @param href
+ * @param $links - links in header
+ */
+function updateTextOnUndefinedLanguagePage(href, $links) {
+    var lang_code = checkForUndefinedLanguagePage(href, $links);
+    if (lang_code) {
+        var subPath = getSubPath(href);
+        changeMissingtextForLanguageCode(lang_code, subPath);
```
Comment (richmahn) [inline js/application.js]: Uppercase T in 'text'

Comment (PhotoNomad0) [inline js/application.js]: Corrected

**Comment by PhotoNomad0** [comment]
@richmahn - ready to merge.  Lambda is now configured on dev.

**Pull request merged**